core_backend/app/utils.py

```diff
@@ -22,6 +22,22 @@
 SECRET_KEY_N_BYTES = 32
 
 
+# To prefix trace_id with project name
+LANGFUSE_PROJECT_NAME = None
+
+if LANGFUSE == "True":
+    langFuseLogger = litellm.utils.langFuseLogger
+    if langFuseLogger is None:
+        langFuseLogger = litellm.integrations.langfuse.LangFuseLogger()
+        LANGFUSE_PROJECT_NAME = (
+            langFuseLogger.Langfuse.client.projects.get().data[0].name
+        )
+    elif isinstance(langFuseLogger, litellm.integrations.langfuse.LangFuseLogger):
+        LANGFUSE_PROJECT_NAME = (
+            langFuseLogger.Langfuse.client.projects.get().data[0].name
+        )
```

Comment:
Or just in case there's trouble getting the project name, I'm not sure if we should do something like

```suggestion
def get_random_string(size: int) -> str:
    """Generate a random string of fixed length."""
    import random
    import string

    return "".join(random.choices(string.ascii_letters + string.digits, k=size))


# To prefix trace_id with project name
LANGFUSE_PROJECT_NAME = None

if LANGFUSE == "True":
    langFuseLogger = litellm.utils.langFuseLogger
    try:
        if langFuseLogger is None:
            langFuseLogger = litellm.integrations.langfuse.LangFuseLogger()
        LANGFUSE_PROJECT_NAME = (
            langFuseLogger.Langfuse.client.projects.get().data[0].name
        )
    except Exception as e:
        LANGFUSE_PROJECT_NAME = get_random_string(8).lower()
```